packages/css/src/create-instance.js

```diff
@@ -1,12 +1,7 @@
 // @flow
```

Comment:
Any reason why this file also declares its own Interpolation?
https://github.com/emotion-js/emotion/pull/3076/files#diff-5d0ca00adf3ee0ff837d19e810ebe3271cfa621468b53c7eafe57f061b782db5R27

Reply:
This can be disregard as this is a mistake on my part